mysql-test/suite/rocksdb/include/type_varchar_endspace.inc

```diff
@@ -42,18 +42,24 @@ insert into t1 values (0, 'ab', 'a-b');
 insert into t1 values (1, 'a ', 'a-space');
 insert into t1 values (2, 'a',  'a');
 insert into t1 values (3, 'a  \t', 'a-tab');
+# force cardinality updates otherwise explains below are non-deterministic
```

Comment:
The comment was really a reminder of where our test is differing from upstream. Seems 5.6 is not as sensitive to the differences of a 4 row table as 5.7 is which causes the instability, but I do get the point, the resulting test is fairly obvious.